core/src/main/resources/hudson/model/UsageStatistics/help-usageStatisticsCollected_pt_BR.html

```diff
@@ -0,0 +1,24 @@
+<div>
+    Saber como Jenkins é utilizado é uma tremenda ajuda guiando-nos no desenvolvimento, especificamente para
+    projetos de código livre onde os usuários são inerentemente difíceis de rastrear. Ao ativar esta opção,
+    Jenkins periodicamente enviará informações sobre o seu uso.
+
+    <p>
+    Especificamente, as informações enviadas serão:
+
+    <ul>
+    <li>A versão do seu Jenkins
+    <li>Para cada master e cada slave, o tipo de Sistema Operacional, e número de executores
+    <li>Os plugins instalados e suas versões
+    <li>Número de jobs por tipo de job no seu Jenkins
+    </ul>
+
+    <p>
+    Essas informaçãos não contem nada o identifique ou nos permita contatá-lo
+    (exceto pelas informações reveladas via HTTP, tal como Endereço IP).
+    Dados das estatísticas de uso submetidas são compartilhados com a comunidade.
+
+    <p>
+    Você pode ler <a href="http://jenkins.361315.n4.nabble.com/request-for-comments-tracking-usage-metrics-td375557.html">
```

Comment:
This paragraph, while in the original English help text as well, should probably be left out – it may have made sense when this feature was initially added, but that thread has been inactive for more than seven years.